manim/mobject/mobject.py

```diff
@@ -1280,16 +1288,23 @@ def rotate(
         angle: float,
         axis: Vector3D = OUT,
```

Comment:
When merging the main branch, this will probably be automatically fixed anyways, but, just in case:
```suggestion
        axis: Vector3DLike = OUT,
```